analyzer/consensus/consensus.go

```diff
@@ -578,8 +578,17 @@ func (m *Main) queueNodeEvents(batch *storage.QueryBatch, data *storage.Registry
 				nodeEvent.Node.SoftwareVersion,
 				0,
 			)
+
+			// Update the node's runtime associations by deleting
+			// all old records and inserting new ones.
+			batch.Queue(queries.ConsensusRuntimeNodesDelete, nodeEvent.Node.ID.String())
+			for _, runtime := range nodeEvent.Node.Runtimes {
+				// XXX: Include other fields here if needed in the future.
```

Comment:
I'm a little confused by the comment. What other fields do you foresee? Especially in the runtime_nodes table ...

Reply:
Any of thesee: https://github.com/oasisprotocol/oasis-core/blob/5b2d859dc05dd2d53fdd1ca110949b8bda179779/go/common/node/node.go#L423-L431